exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java

```diff
@@ -29,141 +35,131 @@ public  Game(){
-			
-			System.out.println(players.get(currentPlayer) 
-					+ "'s new location is " 
-					+ places[currentPlayer]);
-			System.out.println("The category is " + currentCategory());
-			askQuestion();
+
+			moveCurrentPlayer(roll);
 		}
-		
+
+	}
+
+	private void moveCurrentPlayer(int roll) {
+		playerPositions[currentPlayer] = playerPositions[currentPlayer] + roll;
+		if (playerPositions[currentPlayer] > 11) playerPositions[currentPlayer] = playerPositions[currentPlayer] - 12;
+
+		System.out.println(players.get(currentPlayer)
+				+ "'s new location is "
+				+ playerPositions[currentPlayer]);
+		System.out.println("The category is " + currentCategory());
+		askQuestion();
 	}
 
 	private void askQuestion() {
-		if (currentCategory() == "Pop")
+		if (currentCategory() == POP_CATEGORY)
 			System.out.println(popQuestions.removeFirst());
-		if (currentCategory() == "Science")
+		if (currentCategory() == SCIENCE_CATEGORY)
 			System.out.println(scienceQuestions.removeFirst());
-		if (currentCategory() == "Sports")
+		if (currentCategory() == SPORTS_CATEGORY)
 			System.out.println(sportsQuestions.removeFirst());
-		if (currentCategory() == "Rock")
-			System.out.println(rockQuestions.removeFirst());		
+		if (currentCategory() == ROCK_CATEGORY)
+			System.out.println(rockQuestions.removeFirst());
 	}
-	
-	
+
+
 	private String currentCategory() {
-		if (places[currentPlayer] == 0) return "Pop";
-		if (places[currentPlayer] == 4) return "Pop";
-		if (places[currentPlayer] == 8) return "Pop";
-		if (places[currentPlayer] == 1) return "Science";
-		if (places[currentPlayer] == 5) return "Science";
-		if (places[currentPlayer] == 9) return "Science";
-		if (places[currentPlayer] == 2) return "Sports";
-		if (places[currentPlayer] == 6) return "Sports";
-		if (places[currentPlayer] == 10) return "Sports";
-		return "Rock";
+		int playerPosition = playerPositions[currentPlayer];
+		if (playerPosition == 0) return POP_CATEGORY;
+		if (playerPosition == 4) return POP_CATEGORY;
+		if (playerPosition == 8) return POP_CATEGORY;
+		if (playerPosition == 1) return SCIENCE_CATEGORY;
+		if (playerPosition == 5) return SCIENCE_CATEGORY;
+		if (playerPosition == 9) return SCIENCE_CATEGORY;
+		if (playerPosition == 2) return SPORTS_CATEGORY;
+		if (playerPosition == 6) return SPORTS_CATEGORY;
+		if (playerPosition == 10) return SPORTS_CATEGORY;
+		return ROCK_CATEGORY;
 	}
 
-	public boolean wasCorrectlyAnswered() {
+	public boolean isCorrectAnswer() {
 		if (inPenaltyBox[currentPlayer]){
 			if (isGettingOutOfPenaltyBox) {
 				System.out.println("Answer was correct!!!!");
-				purses[currentPlayer]++;
-				System.out.println(players.get(currentPlayer) 
-						+ " now has "
-						+ purses[currentPlayer]
-						+ " Gold Coins.");
-				
-				boolean winner = didPlayerWin();
-				currentPlayer++;
-				if (currentPlayer == players.size()) currentPlayer = 0;
-				
-				return winner;
+				return runTurnRound();

```

Comment:
Great use of extracting from duplication. Just a note on the name, I am not sure what exactly it means